website/docs/r/networkmanager_core_network.html.markdown

```diff
@@ -10,8 +10,6 @@ description: |-
 
 Provides a core network resource.
 
-~> **NOTE on Core Networks and Policy Attachments:** For a given core network, this resource's `policy_document` argument is incompatible with using the [`aws_networkmanager_core_network_policy_attachment` resource](/docs/providers/aws/r/networkmanager_core_network_policy_attachment.html). When using this resource's `policy_document` argument and the `aws_networkmanager_core_network_policy_attachment` resource, both will attempt to manage the core network's policy document and Terraform will show a permanent difference.
```

Comment:
There is a similar note to this in the `aws_networkmanager_core_network_policy_attachment` docs which can also be removed:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/networkmanager_core_network_policy_attachment